spec/concordance_validation/delta_spec.rb

```diff
@@ -9,45 +9,69 @@
 # make test files
 
 RSpec.describe ConcordanceValidation::Delta do
-  let(:delta) { described_class.new("old_concordance.txt", "new_concordance.txt") }
+  def validated_concordance_path(concordance)
+    File.join(Settings.concordance_path, "validated", concordance)
+  end
+
+  let(:old_concordance) { validated_concordance_path "old_concordance.txt" }
+  let(:new_concordance) { validated_concordance_path "new_concordance.txt" }
+  let(:old_concordance_gz) { validated_concordance_path "old_concordance.txt.gz" }
+  let(:new_concordance_gz) { validated_concordance_path "new_concordance.txt.gz" }
+  let(:delta) { described_class.new(old_concordance, new_concordance) }
+  let(:delta_with_gzip) { described_class.new(old_concordance_gz, new_concordance_gz) }
+  let(:adds) { delta.diff_out_path + ".adds" }
+  let(:deletes) { delta.diff_out_path + ".deletes" }
+  let(:old_concordance_data) {
+    <<~END
+      333\t444
+      777\t888
+      555\t666
+      111\t222
+    END
+  }
+  let(:new_concordance_data) {
+    <<~END
+      999\t000
+      555\t666
+      333\t444
+      777\t888
+    END
+  }
+  let(:added_data) { "999\t000\n" }
+  let(:deleted_data) { "111\t222\n" }
 
   before(:each) do
     FileUtils.mkdir_p Settings.concordance_path + "/validated/"
     FileUtils.mkdir_p Settings.concordance_path + "/diffs/"
-    File.write(Settings.concordance_path + "/validated/old_concordance.txt",
-      ["dupe\tdupe", "delete\tdelete"].join("\n"))
-    File.write(Settings.concordance_path + "/validated/new_concordance.txt",
-      ["dupe\tdupe", "add\tadd"].join("\n"))
-  end
-
-  describe "#initialize" do
-    it "opens the old and new concordances for reading" do
-      expect(delta.old_conc.to_a.size).to eq(2)
-      expect(delta.new_conc.to_a.size).to eq(2)
-    end
+    File.write(old_concordance, old_concordance_data)
+    File.write(new_concordance, new_concordance_data)
+    `gzip -c #{old_concordance} > #{old_concordance_gz}`
```

Comment:
I'd use `system` here instead of backticks, since you're not capturing the output.